horovod/spark/common/store.py

```diff
@@ -431,3 +441,40 @@ def _check_url(self, url, prefix, path):
     @classmethod
     def filesystem_prefix(cls):
         return cls.FS_PREFIX
+
+
+class DBFSLocalStore(LocalStore):
+    """Uses DBFS as a store of intermediate data and training artifacts.
+
+    Initialized from a `prefix_path` starts with `/dbfs/...`, see
+    https://docs.databricks.com/data/databricks-file-system.html#local-file-apis.
+
+    Note that DBFSLocalStore cannot be created via Store.create("/dbfs/..."), which will create a
```

Comment:
Maybe a follow-up: If user calls `Store.create("dbfs:/")`, shall we convert it into DBFSLocalStore?

Reply:
Added to backlog.